environment.yml

```
@@ -12,4 +12,4 @@ dependencies:
 - conda-forge::black=19.10b0
 - conda-forge::seaborn=0.9.0
 - pip:
-  - git+https://github.com/cytomining/pycytominer@3138f3bb0eb19a4cab5bd93e2faf47fce953d82f
+  - git+https://github.com/cytomining/pycytominer@3138f3bb0eb19a4cab5bd93e2faf47fce953d82f
```

```suggestion
  - git+https://github.com/cytomining/pycytominer@3138f3bb0eb19a4cab5bd93e2faf47fce953d82f
  
```

sneaky no new line octocon again